gustaf/vertices.py

```diff
@@ -245,7 +245,8 @@ def update_vertices(self, mask, inverse=None):
             )
             # remove all the elements that's not part of inverse
             if check_neg:
-                elements = elements[np.unique(np.where(elements >= 0)[0])]
+                emask = (elements > -1).all(axis=1)
+                elements = elements[emask]
```

Comment:
You are only choosing elements that all have nodes that are present? Others are ignored? Did I understand this correctly? If yes I have no further comments.